content/docs/_index.md

```diff
@@ -2,12 +2,13 @@
 title: "Documentation"
 draft: false
 weight: 1
-aliases:
-- /source-git
 ---
 
-* [Actions](/docs/actions/)
-* [Architecture](/docs/architecture/)
-* [Configuration for packit]({{< ref "/docs/configuration.md" >}})
-* [Packit CLI](/docs/cli/)
-* [Packit Guide](/docs/guide/)
+* [Frequently Asked Questions]({{< ref "faq" >}}) - Whys, Whats, contacts.
+* [Source-git](/docs/source-git/) - Source-git concept and how to work with it.
+* [Onboarding Guide]({{< ref "guide" >}}) - How to start using Packit.
+* [Configuration for packit]({{< ref "configuration" >}}) - Configuration file description.
+* [Testing Farm]({{< ref "testing-farm" >}}) - Guide to Packit's service for running tests.
+* [Packit-as-a-Service]({{< ref "packit-as-a-service" >}}) - GitHub App that helps you continuously ensure that your project work in Fedora OS.
```

Comment:
I'd probably reorder this a bit to:
* [Onboarding Guide]
* [Packit-as-a-Service]
* [Testing Farm]
* [Frequently Asked Questions]
* [Source-git]
* [Configuration for packit]